ddev/src/ddev/cli/dep.py

```diff
@@ -0,0 +1,448 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import asyncio
+import copy
+from collections import defaultdict
+
+import click
+import httpx
+import orjson
+from packaging.markers import Marker
+from packaging.requirements import InvalidRequirement, Requirement
+from packaging.specifiers import InvalidSpecifier, SpecifierSet
+from packaging.utils import canonicalize_name
+from packaging.version import InvalidVersion, Version
+
+# Dependencies to ignore when update dependencies
+IGNORED_DEPS = {
+    'ddtrace',  # https://github.com/DataDog/integrations-core/pull/9132
+    'flup',  # https://github.com/DataDog/integrations-core/pull/1997
+    'dnspython',
+    'pymysql',  # https://github.com/DataDog/integrations-core/pull/12612
+    'foundationdb',  # Breaking datadog_checks_base tests
+    'openstacksdk',  # Breaking openstack_controller tests
+    'pyasn1',  # Breaking snmp tests
+    'pycryptodomex',  # Breaking snmp tests
+    'pysnmp',  # Breaking snmp tests
+    'clickhouse-driver',  # Breaking clickhouse tests
+    'lz4',  # Breaking clickhouse tests
+    'pyodbc',  # Breaking sqlserver tests
+    'psutil',  # Breaking disk tests
+    'aerospike',  # v8+ breaks agent build.
+    'protobuf',  # 3.20.2->4.23.3 breaks kubernetes_state, kube_dns, gitlab and gitlab_runner tests.
+    'service-identity',  # 21.1->23.1 breaks tls tests.
+    'pyvmomi',  # 7->8 breaks vsphere tests.
+    # 4.3->4.4 changes the license field in the package metadata to something our validations cannot handle.
+    'pymongo',
+    # We need pydantic 2.0.2 for the rpm x64 agent build (see https://github.com/DataDog/datadog-agent/pull/18303)
+    'pydantic',
+}
```

Comment:
Your list is not up-to-date, I dropped some of them: https://github.com/DataDog/integrations-core/blob/master/datadog_checks_dev/datadog_checks/dev/tooling/commands/dep.py#L29-L49

Reply:
Thanks for the heads-up! I've synced it with the original ignorelist now.